docs/tutorials/issue-tracker/07-operator-watcher.md

```diff
@@ -172,23 +192,18 @@ func (s *IssueWatcher) PrometheusCollectors() []prometheus.Collector {
 	return []prometheus.Collector{s.statsGauge}
 }
 ```
-Easy! Now all we need to do is register it with our operator so it can expose it. The `simple.Operator` provides a method we can call in our `main` function, which we can add right after the part that creates and adds the watcher with `runner.WatchKind` (`runner` is the variable name for our `simple.Operator`):
+Easy! Now we need to do is register the Collector.
 ```go
-// Wrap our resource watchers in OpinionatedWatchers, then add them to the controller
-issueWatcher, err := watchers.NewIssueWatcher()
-if err != nil {
-	logging.DefaultLogger.With("error", err).Error("Unable to create IssueWatcher")
-	panic(err)
-}
-err = runner.WatchKind(issue.Kind(), issueWatcher, simple.ListWatchOptions{
-	Namespace: resource.NamespaceAll,
-})
-if err != nil {
-	logging.DefaultLogger.With("error", err).Error("Error adding Issue watcher to controller")
-	panic(err)
-}
-// Register prometheus collectors from the watcher
-runner.RegisterMetricsCollectors(issueWatcher.PrometheusCollectors()...)
+// pkg/app/app.go
+
+func New(cfg app.Config) (app.App, error) {
+	issueWatcher, err := watchers.NewIssueWatcher()
+	if err != nil {
+		return nil, fmt.Errorf("unable to create IssueWatcher: %w", err)
+	}
+
+	prometheus.MustRegister(issueWatcher.PrometheusCollectors()...)
```

Comment:
This is a bodge that worked for me, i suspect there's a better way to register this collector, but I couldn't see anything useful in `simple.App`